src/MatchesSnapshots.php

```diff
@@ -130,23 +130,25 @@ protected function getFileSnapshotDirectory(): string
      * matched.
      *
      * Override this method it you want to use a different flag or mechanism
-     * than `-d --update-snapshots`.
+     * than `-d --update-snapshots` or `UPDATE_SNAPSHOTS=true` env var.
      */
     protected function shouldUpdateSnapshots(): bool
     {
-        return in_array('--update-snapshots', $_SERVER['argv'], true);
+        return in_array('--update-snapshots', $_SERVER['argv'], true)
```

Comment:
For improved readability, we could break up this conditional and use early returns

Reply:
See my comments below about that.